infrateam_first_responder.md

```diff
@@ -275,8 +206,84 @@ Prioritization is the responsibility of management, not the responsibility of th
   * How many digital resources are affected?
   * Is it blocking time sensitive work?
 
-# What if first responder isn't available?
+### Should a first responder do work cycle work?
+After completing the specific responsibilities listed above and any prioritized production issues, a developer may choose at their discretion to (1) perform some of the general responsibilities listed above; (2) work on other issues from the production priorities board; and/or (3) do work cycle work.
+
+### What if first responder isn't available?
 
 The idea is for the first responder to be "interruptible" for production problems during his/her week of coverage.  If there are significant blocks of time when this isn't true (e.g. ½ day meeting with no access to slack or email), or if life happens (illness, family emergency, etc.), hopefully the first responder can arrange for coverage.  ("I'll take one of your days if you can cover Tues for me").  Please notify `#dlss-infrastructure` Slack channel of changes.
 
 When the first responder can't arrange for coverage, the default would be for the "on deck" responder to become first responder. The "on deck" person is the first responder for the following week.
+
+### Swapping shifts
+If you are not available for a shift, it is your responsibility to find a swap, not current first responder.
+
+
+DEPRECATE (to be removed after team review):
+## Help vet systemic puppet changes
+
+_Experiment starting July 2021_
+
+Ops will give a bit of advance notice to the infrastructure team before deploying systemic puppet changes (e.g. refactor/cleanup that touches many apps, networking changes, etc).  First responder will work with ops to determine whether testing is needed, and if so, what testing is appropriate, based on what we worry might break.  Running `infrastructure-integration-tests`  against stage after the puppet changes have been deployed there is a good default practice.  Don't hesitate to ask for help planning or performing the testing if needed.
+
+
+### Required Additional Deploys
+
+There are applications that need to be deployed separately (i.e., not using `sdr-deploy`).
+
+#### Cloud Projects
+
+* **dlme-transform**: Check that [CircleCI](https://app.circleci.com/pipelines/github/sul-dlss/dlme-transform) published the latest image after dependency updates were merged. This image is pulled afresh for every data transform. More details in [README](https://github.com/sul-dlss/dlme-transform/#deploying) and [DevOpsDocs](https://github.com/sul-dlss/DevOpsDocs/blob/master/projects/dlme/operations-concerns.md#deployment-info).
```

Comment:
Should we still be checking CI to make sure that `main` builds are 💚 as part of dep update workflow, or is that no longer necessary? cc: @aaron-collier